### DISCUSSION TITLE
直播弹幕发送栏会触发edge自动表单

### DISCUSSION THREAD

**Discussion by whynotjinx** [the1812/Bilibili-Evolved#2994] [功能建议 / Ideas]
如题，选中常驻弹幕发送栏会弹出自动表单；
并且不知道是不是我个人问题，发送栏消失的太快了，来不及打完几个字就会消失

**Comment by the1812** [comment]
自动表单我改改, 应该可以关掉

> 发送栏消失的太快了，来不及打完几个字就会消失

win11下中文输入法打字时会被判定成未聚焦, 导致下面整条栏自动隐藏掉了

**Reply by xxxqq55** [reply]
我现在还是会在打中文时，发送栏就自己消失了，这是啥情况？而且输入一个拼音就直接触发了
